### PR TITLE
Fix some mislead information when setting up a cluster from scratch using CNI with flannel

### PR DESCRIPTION
**Pull request by smartwjw**

### CODE REVIEW

**Comment by bradtopol** [comment]
@cmluciano Can you do a tech review of this. If it looks good just add a /lgtm.
Thanks!

File: content/en/docs/setup/scratch.md [hunk 1]
```diff
@@ -350,7 +350,7 @@ ip link delete docker0
 The way you configure docker will depend in whether you have chosen the routable-vip or overlay-network approaches for your network.
 Some suggested docker options:
 
-  - create your own bridge for the per-node CIDR ranges, call it cbr0, and set `--bridge=cbr0` option on docker.
+  - create your own bridge for the per-node CIDR ranges, call it cbr0, and set `--bridge=cbr0` option on docker，you **DON'T** need to do this if you are using kubernetes CNI with Flannel, since kubelet will create `cni0` for you automatically.
```
Comment (cmluciano) [inline content/en/docs/setup/scratch.md]: This is true, but doesn't this also happen for other CNI plugins too?

File: content/en/docs/setup/scratch.md [hunk 2]
```diff
@@ -766,10 +767,13 @@ Template for controller manager pod:
 Flags to consider using with controller manager:
 
  - `--cluster-cidr=`, the CIDR range for pods in cluster.
+ - `--allocate-node-cidrs=true`, allow CIDRs allocated for Node to make CNI plugins work properly.
```
Comment (cmluciano) [inline content/en/docs/setup/scratch.md]: Can we just add this context to the option on the next line? This also is not enough for the [CNI use-case](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni) if do not include `--cluster-cidr=<cidr>`

File: content/en/docs/setup/scratch.md [hunk 3]
```diff
@@ -766,10 +767,13 @@ Template for controller manager pod:
...
  - `--allocate-node-cidrs=`, if you are using `--cloud-provider=`, allocate and set the CIDRs for pods on the cloud provider.
  - `--cloud-provider=` and `--cloud-config` as described in apiserver section.
  - `--service-account-private-key-file=/srv/kubernetes/server.key`, used by the [service account](/docs/user-guide/service-accounts) feature.
- - `--master=127.0.0.1:8080`
+ - `--root-ca-file=/srv/kubernetes/ca.pem`, used by the [service account](/docs/user-guide/service-accounts) feature.
+ - `--master=127.0.0.1:8080`.
```
Comment (cmluciano) [inline content/en/docs/setup/scratch.md]: why hardcode this to localhost?

File: content/en/docs/setup/scratch.md [hunk 4]
```diff
@@ -766,10 +767,13 @@ Template for controller manager pod:
 Flags to consider using with controller manager:
 
  - `--cluster-cidr=`, the CIDR range for pods in cluster.
+ - `--allocate-node-cidrs=true`, allow CIDRs allocated for Node to make CNI plugins work properly.
  - `--allocate-node-cidrs=`, if you are using `--cloud-provider=`, allocate and set the CIDRs for pods on the cloud provider.
  - `--cloud-provider=` and `--cloud-config` as described in apiserver section.
  - `--service-account-private-key-file=/srv/kubernetes/server.key`, used by the [service account](/docs/user-guide/service-accounts) feature.
- - `--master=127.0.0.1:8080`
+ - `--root-ca-file=/srv/kubernetes/ca.pem`, used by the [service account](/docs/user-guide/service-accounts) feature.
```
Comment (cmluciano) [inline content/en/docs/setup/scratch.md]: this is fine but we should also remove the `--service-account-private-key-file` if that is now removed

**Comment by zacharysarah** [comment]
@cmluciano 👋 Thanks for the review!

@smartwjw Please feel free to reopen this PR when you're ready to address [review feedback](https://github.com/kubernetes/website/pull/9819#pullrequestreview-147696083).

**Pull request closed without merging**